lib/pagination/paginate.js

```diff
@@ -0,0 +1,30 @@
+const Adapter = require('./adapter');
+const config = require('./config');
+const logger = require('../../test/app/logger/index');
```

Comment:
Don't use logger of test folder, you could receive some logging option in parameters